caramel/config.py

```diff
@@ -71,3 +85,22 @@ def _get_config_value(
             env_var,
         )
     return result
+
+
+def get_ca_cert_key(arguments, settings=None, required=True):
```

Comment:
This doesn't get a cert and key, but paths.

Also, required=True as a keyword argument is never used by any caller? Should probably be removed.

Reply:
also, other code sets a type on "arguments", I'd suggest doing it here as well.

Reply:
Not all tools use a cert and key strictly, caramel_tools --list doesnt use a cert at all if I'm not mistaken(even though it checks for one and will arbitrarily crash if there is none provided) for example. I understand why --list is a flag for tools, it saves bloated code and is a nice collection of similar-ish functionality,I just thought this was a kind of future-proofing that doesn't add any major complication the way the function is currently used.